README.maintainer.md

```diff
@@ -1,16 +1,19 @@
-# Directions for overriding the C libraries (flint)
+# Directions for overriding the C libraries
 
-In `.julia/artifacts` you should have or make a file `Overrides.toml` with the following
-entry (replace `/usr/local` with the location of your installation)
+## FLINT
+
+In `.julia/artifacts` you should have or make a file `Overrides.toml` with the
+following entry (replace `/usr/local` with the location of your installation)
```

Comment:
The `Overrides.toml` is not FLINT specific, but C library specific. Why move it to the FLINT subsection?

Reply:
Perhaps the original heading was unclear, as it had "flint" in it.